models/movement.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const mongoose = require('mongoose');
+
+const movementSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  description: {
+    type: String,
+    required: true,
+    max: 200
+  },
+  startYear: String,
+  endYear: String,
+  ancestor: String
```

Comment:
'ancestor' should be a reference to another Movement; this type should be ObjectId.
See notes from `express-api` lesson on `_owner`.